torch/fx/passes/infra/partitioner.py

```diff
@@ -42,15 +42,7 @@ def __init__(self,
         self.allows_single_node_partition = allows_single_node_partition
 
     def __is_node_supported(self, node: Node) -> bool:
-        return (
-            self.operator_support.is_node_supported(dict(self.graph_module.named_modules()), node)
-            and
-            # reject 'getitem' node since they are special cased in partitioning.
-            (
-                node.op != "call_function" or
-                _get_qualified_name(node.target) != "_operator.getitem"    # type: ignore[arg-type]
-            )
-        )
+        return self.operator_support.is_node_supported(dict(self.graph_module.named_modules()), node)
```

Comment:
BTW, this is not safe.

You'll have `getitem` node absorbed by its consumer and later hitting an assert somewhere in the merge_single_node function. I think you'll be failing one of the `getitem` special handling tests we have.